trucks/functions/update-truck/main.go

```diff
@@ -51,6 +51,14 @@ func apiGatewayHandler(ctx context.Context, request events.APIGatewayProxyReques
 		truck.Year = year
 	}
 
+	if body.Get("completed_trips") != "" {
```

Comment:
I think this action should not be legal, we are just let it increment naturally

Reply:
so you mean to remove this (and only letting the change occur in the event that increments the value) or only allow sequential increment?